README.md

```diff
@@ -8,26 +8,61 @@ Loads CSV records from file, and maps them to FHIR resources.
 
 ## Quickstart CLI
 
+### OS X / Linux
 ```shell
 # clone the repo
 git clone https://github.com/LinuxForHealth/CsvToFHIR.git
-cd csvtofhir
+cd CsvToFHIR
 
 # create virtual environment and create an "editable" install
-python3 -m venv venv && source venv/bin/activate && python3 -m pip install --upgrade pip setuptools
+python3 -m venv venv --clear && \
+        source venv/bin/activate && \
+        python3 -m pip install --upgrade pip setuptools wheel
+        
 python3 -m pip install -e .[dev]
 # run tests
 python3 -m pytest
 ```
 
-The CSVToFHIR CLI supports:
+### Windows Setup
+Launch the Windows Command and "Run as Administrator"
+```shell
+# clone the repo
+git clone https://github.com/LinuxForHealth/CsvToFHIR.git
+cd CsvToFHIR
+
+# create the virtual environment (may take some time to complete)
+python3 -m venv venv --clear
+.\venv\Scripts\activate.bat
+python3 -m pip install --upgrade pip setuptools wheel
+# integrate the local development environment with the virtual environment
+python3 -m pip install -e .[dev]
```

Comment:
put "'s around the .[dev]

Also for all these commands on windows i just used python (not python3)

Reply:
Thanks! The command will differ based on the environment and may vary based on how Python is installed on the system. I was able to use the command above on Windows 10 Pro without any issues. 

In regards to the executable name "python" is "symlinked" (probably a shortcut) to python3 on Windows.